fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -590,6 +591,9 @@ private void initAggregateBuiltins() {
                 Lists.newArrayList(Type.PERCENTILE), Type.PERCENTILE, Type.PERCENTILE,
                 false, false, false));
 
```

Comment:
What's the behavior if the arg type is array_int, array_varchar？

Reply:
Because conditions' result type must be boolean(uint8), and we call get_uint8 on Datum, so it probably incompatible with array_int, array_varchar.